File: applied/7_i_just_called_to_say_i_love_you/answer/1.c
// 5個の整数を入力し、それらの平均を出力しよう
// 平均は小数第一位まで表示してください

#include <stdio.h>
int main(void){
    int input, i;
    float sum;
    
    for (i=0, avg=0; i<5; i++) {
        scanf("%d", &input);
        sum += input;
    }
    
    printf("%.1f\n", sum/5);
}

File: applied/07_i_just_called_to_say_i_love_you/answer/1.java
// 5個の整数を入力し、それらの平均を出力しよう
// 平均は小数第一位まで表示してください

import java.util.*;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner scan = new Scanner(System.in);
        float sum = 0;
        
        for (int i=0; i<5; i++) {
            int input = scan.nextInt();
            sum += input;
        }
        
        System.out.printf("%.1f\n", sum/5);
    }
}
